### ISSUE TITLE
Double Log-output - why?

### ISSUE THREAD

**Issue by beniroquai** [ImSwitch/ImSwitch#217]
@jacopoabramo for some reasons all logs come duplicated. One in colour, one without. I remember that at one point I touched the `tryInheritParent` setting, but I cannot find differences between old ImSwitch master code and mine. Any ideas? (CC @kasasxav 😉)

<img width="505" alt="image" src="https://github.com/user-attachments/assets/4644aad2-e0e0-49b3-aff1-ac865e7c5a61">

**Comment by jacopoabramo** [comment]
Yes I notice this pretty often as well. Just out of curiosity how do you launch ImSwitch in this scenario?

**Comment by beniroquai** [comment]
I can launch it in any way e.g. 
`python3 -m imswitch`, 
`imswitch` 
.. .
The result is the same. Maybe some settings that went off with the coloured logs?
